### PR TITLE
add hw6

### PR DESCRIPTION
**Pull request by yanaPy**

### CODE REVIEW

File: hw6/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package hw6;
```
Comment (AndrewSkoba) [inline hw6/Calculator.java:1]: Pls, group into packages

File: hw6/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+public class Calculator {
+
+    static int addition(int num, int num2) {
```
Comment (AndrewSkoba) [inline hw6/Calculator.java:5]: Would be better to use public. It is looks like utility methods

File: hw6/Main.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package hw6;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        System.out.println(Calculator.addition(10, 1));
```
Comment (AndrewSkoba) [inline hw6/Main.java:7]: Import can be static here

File: hw6/Calculator.java (hunk 1 above)
Comment (yanaPy) [reply hw6/Calculator.java:1]: done

File: hw6/Calculator.java (hunk 2 above)
Comment (yanaPy) [reply hw6/Calculator.java:5]: done

File: hw6/Main.java (hunk 3 above)
Comment (yanaPy) [reply hw6/Main.java:7]: done

**Pull request closed without merging**